datafusion/physical-expr/src/equivalence/properties.rs

```diff
@@ -22,7 +22,9 @@ use std::slice::Iter;
 use std::sync::Arc;
 use std::{fmt, mem};
 
-use crate::equivalence::class::{const_exprs_contains, AcrossPartitions};
+use crate::equivalence::class::{
```

Comment:
Looking through the properties.rs file, it seems like it is getting huge -- I will file a follow on ticket about that